omod-1.12/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_12/OrderGroupResource1_12.java

```diff
@@ -155,18 +155,16 @@ public Model getGETModel(Representation rep) {
 	
 	@Override
 	public Model getCREATEModel(Representation representation) {
-		ModelImpl model = new ModelImpl().property("patient", new StringProperty().example("uuid"))
+		return new ModelImpl().property("patient", new StringProperty().example("uuid"))
 		        .property("encounter", new StringProperty().example("uuid"))
 		        .property("orders", new ArrayProperty(new RefProperty("#/definitions/OrderCreate")))
 		        .property("orderSet", new StringProperty().example("uuid"));
-		return model;
 	}
 	
 	@Override
 	public Model getUPDATEModel(Representation rep) {
-		ModelImpl model = new ModelImpl().property("orders",
+		return new ModelImpl().property("orders",
 		    new ArrayProperty(new RefProperty("#/definitions/OrderCreate")));
```

Comment:
```suggestion
		return new ModelImpl().property("orders", new ArrayProperty(new RefProperty("#/definitions/OrderCreate")));
```
Why not in one line?

Reply:
Hey @sacull 
Setting the IDE as pre-described by this wiikik https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+Eclipse does automatically  code formatting on save action. This action formats the source codes as per OpenMRS standards!

Reply:
It looks like you might be right. I was based on: 
> Long lines. Lines should not exceed 125 characters.
source: https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Others

> <setting id="org.eclipse.jdt.core.formatter.comment.line_length" value="100"/>
source: https://github.com/openmrs/openmrs-core/blob/master/tools/src/main/resources/eclipse/OpenMRSFormatter.xml#L200

So one document says 125 characters, another 100 characters, so we can assume that you are right and we can leave your version. ;)

Reply:
Is there any other changes that I need to address concerning this ticket?
cc: @sacull

Reply:
https://github.com/openmrs/openmrs-module-webservices.rest/pull/472#issuecomment-728675452 ;)

Reply:
Hey @sacull  have addressed the comment as per say